spec/support/mailchimp_helper.rb

```diff
@@ -0,0 +1,66 @@
+module MailchimpHelper
+  # Helper function to create a Gibbon request
+  def mailchimp_create_request
+    Gibbon::Request.new(
+      api_key: ENV['MAILCHIMP_API_KEY']
+    )
+  end
+
+  # Get all members in list
+  def mailchimp_get_members(list_id)
+    mailchimp_create_request
+      .lists(list_id)
+      .members
+      .retrieve
+  end
+
+  # Check if a member exists in a list by email
+  def mailchimp_member_exists?(email, list_id)
```

Comment:
This method is doing a bit too much, can we move the temporary variables into private methods please?